packages/icons/src/icons/twitter-x.js

```diff
@@ -0,0 +1,28 @@
+import React, { forwardRef } from "react"
+import p from "prop-types"
+export const TwitterX = forwardRef(function TwitterX(props, ref) {
+  const { color = "currentColor", size = 24, ...rest } = props
+  return (
+    <svg {...rest} ref={ref} stroke={color} width={size} height={size}>
+      <path
+        class="st0"
+        d="M21.3 21.1 9.9 2.9 2.7 2.9 14.1 21.1zM2.7 21.1 9.9 14.5M14.1 9.5 21.3 2.9"
+      ></path>
```

Comment:
@stassribnyi Assuming this was autogenerated, but I don't think we need the `class` here. I think we can keep this as a single line to match the existing components.
```suggestion
      <path d="M21.3 21.1 9.9 2.9 2.7 2.9 14.1 21.1zM2.7 21.1 9.9 14.5M14.1 9.5 21.3 2.9"></path>
```

Reply:
@jgrubard, it is indeed auto-generated. First, it introduces this class to the SVG icon when the minification step passes. Then, it uses the minified SVG in the .js file. I can remove it, but once anyone else runs `yarn compile`, it will be back again. The only way to get rid of this is to modify the build.js file, but I am not sure of the consequences.